src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Transform.cs

```diff
@@ -204,7 +204,7 @@ public Hashtable PropagatedNamespaces
                 CanonicalXmlNodeList namespaces = null;
                 if (reference != null)
                     namespaces = reference._namespaces;
-                else if (signedXml._context != null)
+                else if (signedXml?._context != null)
```

Comment:
If the tests found a product bug, but the product bug also exists in .NET Framework, we should leave the bug for now and open a new issue to track it (so that the fix gets marked port-consider and it's obvious what part we're port-considering).